src/main/java/com/bandwidth/sqs/consumer/Consumer.java

```diff
@@ -38,8 +36,8 @@
 
 public class Consumer {
```

Comment:
This is going to be added in a PR I'm working on right now... will be opened soon.